public/app/features/manage-dashboards/components/ImportDashboardForm.tsx

```diff
@@ -0,0 +1,185 @@
+import React, { FormEvent, PureComponent } from 'react';
+import { connect } from 'react-redux';
+import { Forms } from '@grafana/ui';
+import { dateTime } from '@grafana/data';
+import { FolderPicker } from 'app/core/components/Select/FolderPicker';
+import DataSourcePicker from 'app/core/components/Select/DataSourcePicker';
+import { changeDashboardTitle, resetDashboard, saveDashboard, changeDashboardUid } from '../state/actions';
+import { DashboardSource } from '../state/reducers';
+import { StoreState } from '../../../types';
+import { ButtonVariant } from '@grafana/ui/src/components/Forms/Button';
```

Comment:
Let's not use path imports. You can add https://github.com/grafana/grafana/pull/22395/files#diff-d0482cb62e23d7e99d9af6954b4acfc1R143 to get ButtonVariant from @grafana/ui

Reply:
This is a whops by Webstorm

Reply:
yeah, been there. It imports by path when there is no export in the package available